chef/cookbooks/monasca/libraries/helper.rb

```diff
@@ -32,6 +32,13 @@ def self.api_public_url(node)
     "#{protocol}://#{host}:#{port}/v2.0"
   end
 
+  def self.log_api_public_url(node)
+    host = monasca_public_host(node)
+    protocol = node[:monasca][:log_api][:ssl] ? "https" : "http"
```

Comment:
I think we don't have **SSL** support at this point.
There are 2 things, at least, that we should do to have it:

* pass that information to ansible role
* make ansible role capable of configuring ssl

Are we sure we have that ?